superset-frontend/src/components/ListView/types.ts

```diff
@@ -55,11 +55,14 @@ type FilterOperator =
   | 'all_text'
   | 'chart_all_text'
   | 'dataset_is_null_or_empty'
-  | 'between';
+  | 'between'
+  | 'dashboard_is_fav'
+  | 'chart_is_fav';
```

Comment:
I'd prefer that we don't abbreviate `favorite` throughout this PR, as using the full word is clearer, and with modern IDEs and autocomplete, it shouldn't make much of a difference either way. Thanks!

Reply:
That would require a backend change as this corresponds to a custom filter on the api
https://github.com/apache/incubator-superset/blob/7dac150dc6f9279c93a832e509aaa89bd35412bb/superset/charts/filters.py#L53

Reply:
Does anyone feel particularly passionate about changing the frontend AND backend over this, or can we let it slide for now?

Reply:
yikes, keep it as is for now I guess..

Maybe we can make this breaking change prior to 1.0 though

Reply:
See PR https://github.com/apache/incubator-superset/pull/12075 which tackles this. Though Cypress is stuck there for some reason